testsuite/bond_tests.py

```diff
@@ -1,36 +1,36 @@
 # Import pyMBE and other libraries
 import pyMBE
 import numpy as np
-        
+
 # Create an instance of pyMBE library
 pmb = pyMBE.pymbe_library(SEED=42)
 
 def check_bond_setup(bond_object, input_parameters, bond_type):
-        """
-        Checks that pyMBE sets up a bond object correctly.
-
-        Args:
-                bond_object(`espressomd.interactions`): instance of a espresso bond object.
-                input_parameters(`dict`): dictionary with the parameters for the bond potential.
-                bond_type(`str`): label identifying the bond type
```

Comment:
When visualizing the diff on GitHub, be sure to click on "Hide whitespaces" under the gear icon to remove clutter. In the terminal, use `git show --color-words HEAD^` to remove whitespace (also works with `git diff`). In some files, you might see extra `^M` at the end of lines: some files on the main branch were saved with Windows line endings (`CRLF`) instead of Unix line endings (`LF`). I chose not to change line endings to Unix in this PR, otherwise the diff would have become unreadable. Maybe someone else can volunteer to go through all scripts and change line endings in a separate PR, using [`dos2unix`](https://packages.ubuntu.com/noble/dos2unix) or text editors like `gedit`. Right now copy-pasting a code sample from a Windows file into a Unix file can lead to extra newline characters appearing, depending on the text processor you are using. In fact, some of the scripts in pyMBE have double newlines in a few code blocks, probably because of copy-paste issues.

Reply:
Also, while changing line endings, one could also apply automatic formatting with `autopep8` to remove trailing whitespace characters. One would have to override `autopep8`'s max line length to avoid wrapping the code to 80 characters per line.